src/NuGet.Core/NuGet.Configuration/Utility/SettingsUtility.cs

```diff
@@ -155,6 +159,59 @@ public static string GetGlobalPackagesFolder(ISettings settings)
             return path;
         }
 
+        /// <summary>
+        /// Read fallback folders from the environment variable or from nuget.config.
+        /// </summary>
+        public static IReadOnlyList<string> GetFallbackPackageFolders(ISettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            var paths = new List<string>();
+
+            var envValue = Environment.GetEnvironmentVariable(FallbackPackagesFolderEnvironmentKey);
+
+            if (string.IsNullOrEmpty(envValue))
+            {
+                // read config values
+                paths.AddRange(GetFallbackPackageFoldersFromConfig(settings));
+            }
+            else
+            {
+                paths.AddRange(envValue.Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries));
```

Comment:
I think it could be in the nuget.exe tests, for the rest I don't think it's possible since they run in parallel